gunicorn/config.py

```diff
@@ -1384,6 +1384,18 @@ class StatsdPrefix(Setting):
     .. versionadded:: 19.2
     """
 
+class StatsdRate(Setting):
+    name = "statsd_rate"
+    section = "Logging"
+    cli = ["--statsd-rate"]
+    meta = "STATSD_RATE"
+    default = "1.0"
+    validator = validate_string
```

Comment:
I should add that too since this PR is actually using a string but neglecting to cast it.